common/utils/src/main/resources/error/error-classes.json

```diff
@@ -6705,6 +6705,259 @@
       "Failed to get block <blockId>, which is not a shuffle block"
     ]
   },
+  "_LEGACY_ERROR_TEMP_3050" : {
+    "message" : [
+      "Cannot modify the value of a static config: <k>"
+    ]
+  },
+  "_LEGACY_ERROR_TEMP_3051" : {
+    "message" : [
+      "When resolving <u>, fail to find subplan with plan_id=<planId> in <q>"
+    ]
+  },
+  "_LEGACY_ERROR_TEMP_3052" : {
```

Comment:
Can we use `UNSUPPORTED_FEATURE`?
- https://spark.apache.org/docs/latest/sql-error-conditions-unsupported-feature-error-class.html#unsupported_feature-error-class